### PR TITLE
Box plot stories

### PR DESCRIPTION
**Pull request by kwongz**
### Checklist

- [x] I have added a [changeset](https://github.com/evidence-dev/evidence/blob/main/CONTRIBUTING.md#adding-a-changeset)

### CODE REVIEW

**Comment by kwongz** [comment]
@zachstence I've tried re-running the chromatic test a few times but the same error is occurring. Also updated the stories argTypes format.

**Comment by zachstence** [comment]
> @zachstence I've tried re-running the chromatic test a few times but the same error is occurring. Also updated the stories argTypes format.

I think merging in the `next` branch should fix it. I just did that.

File: packages/ui/core-components/src/lib/unsorted/viz/box/BoxPlot.stories.svelte [hunk 1]
```diff
@@ -0,0 +1,193 @@
...
+			},
+			xFmt: {
+				control: 'text'
+			},
+			seriesColors: {
+				control: 'object'
+			},
+			swapXY: {
+				control: 'boolean',
+				options: [true, false]
```
Comment (zachstence) [inline packages/ui/core-components/src/lib/unsorted/viz/box/BoxPlot.stories.svelte:49]: I don't think you need to specify `options` for a boolean control

File: packages/ui/core-components/src/lib/unsorted/viz/box/BoxPlot.stories.svelte [hunk 2]
```diff
@@ -0,0 +1,193 @@
...
+		intervalTop: 'max_fare',
+		yFmt: 'usd0'
+	}}
+	let:args
+>
+	<BoxPlot {...args} data={flightData} />
+</Story>
+
+<Story
+	name="swapXY=true colors"
```
Comment (zachstence) [inline packages/ui/core-components/src/lib/unsorted/viz/box/BoxPlot.stories.svelte:165]: Why does this have "colors" in the name?

File: packages/ui/core-components/src/lib/unsorted/viz/box/BoxPlot.stories.svelte [hunk 3]
```diff
@@ -0,0 +1,193 @@
...
+	args={{
+		color: 'color',
```
Comment (zachstence) [inline packages/ui/core-components/src/lib/unsorted/viz/box/BoxPlot.stories.svelte:167]: `color` isn't a valid color, so this does nothing

**Review by zachstence — CHANGES_REQUESTED** [review]

File: packages/ui/core-components/src/lib/unsorted/viz/box/BoxPlot.stories.svelte (hunk 3 above)
Comment (kwongz) [reply packages/ui/core-components/src/lib/unsorted/viz/box/BoxPlot.stories.svelte:167]: color refers to the name of column type defined in the sql query.
image
image

	
Sorry poor naming convention

File: packages/ui/core-components/src/lib/unsorted/viz/box/BoxPlot.stories.svelte (hunk 2 above)
Comment (kwongz) [reply packages/ui/core-components/src/lib/unsorted/viz/box/BoxPlot.stories.svelte:165]: story has horizontal boxplot + colored boxes

File: packages/ui/core-components/src/lib/unsorted/viz/box/BoxPlot.stories.svelte [hunk 4]
```diff
@@ -0,0 +1,214 @@
...
+			eChartsOptions: {
+				control: 'object'
+			},
+			seriesOptions: {
+				control: 'object'
+			},
+			printEchartsConfig: {
+				control: 'boolean',
+				options: ['true', 'false']
+			}
```
Comment (ItsMeBrianD) [inline packages/ui/core-components/src/lib/unsorted/viz/box/BoxPlot.stories.svelte:105]:
```suggestion
			printEchartsConfig: {
				control: 'boolean',
			}
```

**Comment by zachstence** [comment]
@kwongz The Chromatic screenshots for the stories appear empty. Is this expected?

image

**Comment by kwongz** [comment]
image

Boxplot require refresh to render

**Review by zachstence — APPROVED** [review]
Merging this now. We'll investigate the stories being blank separately

**Pull request merged**